projects/canopy/src/lib/forms/validation/validation.component.scss

```
@@ -1,4 +1,6 @@
-@import '../../../styles/mixins.scss';
+@import '../../../styles/mixins';
+// The below import will be removed in the next breaking change
```

```suggestion
/*
 The below import will be removed in the next breaking change. It is currently included to ensure
 the same scss entities are exported. @deprecated
*/
```
Not the best wording but a bit more context might help the next person, added the @deprecated tag too just to help track it.

---

Updated 👍